client/lib/products-values/index.js

```diff
@@ -359,6 +364,25 @@ export function getDomain( product ) {
 	return product.meta;
 }
 
+export function getJetpackProductDisplayName( product ) {
+	product = formatProduct( product );
+	assertValidProduct( product );
+
+	return JETPACK_PRODUCT_DISPLAY_NAMES?.[ product?.productSlug ];
+}
+
+export function getJetpackProductTagLine( product ) {
```

Comment:
Since `tagline` is a single word, we'll also have to rename `TagLine` to `Tagline` here.